src/pages/index.tsx

```diff
@@ -16,7 +63,12 @@ export default function Home() {
       <main className={styles.main}>
         <div className={styles.center}>
           <div className={styles.description}>
-            <p>Coming soon!</p>
+            Profile Picture URL: {profile?.gitHub?.profile?.avatarUrl}<br/><br/>
+            Name: {profile?.gitHub ? profile?.gitHub?.profile?.name :  "loading..."}<br/><br/>
```

Comment:
Maybe show "loading..." instead of the whole `div` content?

Reply:
yes, that looks better